# HG changeset patch
# User David Bateman <dbateman@free.fr>
# Date 1205248123 14400
#      Tue Mar 11 11:08:43 2008 -0400
# Branch release-3-0-x
# Node ID 738ca2f7c83e100f4b2a90de708fb8b2e617f5ae
# Parent  b7a30502f0c965e8988dd85d4a7da29d4260d7cd
run-octave.in: fix typo

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,12 @@
+2008-03-11  David Bateman  <dbateman@free.fr>
+
+	* run-octave.in: Fix typo.
+
 2008-02-21  John W. Eaton  <jwe@octave.org>
 
 	* examples/Makefile.in (install install-strip): Ignore errors
 	installing octave.desktop file.
 
 2008-02-15  John W. Eaton  <jwe@octave.org>
 
 	* configure.in (*-*-freebsd*, *-*-openbsd*): Don't set SH_LD.
diff --git a/run-octave.in b/run-octave.in
--- a/run-octave.in
+++ b/run-octave.in
@@ -45,17 +45,17 @@ d4_list=`$FIND "$d4" -type d -a ! \( \( 
 
 d1_path=`echo "$d1_list" | $AWK '{ t = (s $0); s = t; } END { sub (/:$/, "", s); print s; }'`
 d2_path=`echo "$d2_list" | $AWK '{ t = (s $0); s = t; } END { sub (/:$/, "", s); print s; }'`
 d3_path=`echo "$d3_list" | $AWK '{ t = (s $0); s = t; } END { sub (/:$/, "", s); print s; }'`
 d4_path=`echo "$d4_list" | $AWK '{ t = (s $0); s = t; } END { sub (/:$/, "", s); print s; }'`
 
 LOADPATH="$d1_path:$d2_path:$d3_path:$d4_path"
 IMAGEPATH="$top_srcdir/scripts/image"
-INFOFILE="$builddir/doc/interperter/octave.info"
+INFOFILE="$builddir/doc/interpreter/octave.info"
 
 if [ $# -gt 0 ]; then
   if [ "x$1" = "x-g" ]; then
     driver="gdb --args"
     shift
   elif [ "x$1" = "x-valgrind" ]; then
     driver="valgrind --tool=memcheck"
     shift
